k2/csrc/ragged.h

```diff
@@ -316,8 +316,29 @@ struct Ragged {
     if (!is.eof() || is.fail())
       K2_LOG(FATAL) << "Failed to construct Ragged array from string: " << src;
   }
-  // Construct from context and string.  This uses delegating constructors, (a
-  // c++11 feature), and an explicitly constructed Ragged<T>
+
+  explicit Ragged(const std::vector<std::vector<T>> &vecs) {
+    std::vector<std::vector<int32_t>> row_splits;
+    std::vector<T> elems;
+    row_splits.push_back(std::vector<int32_t>(1, 0));
```

Comment:
If I understand correctly, this function will just create a ragged tensor with `NumAxes() == 2`,  then the below code is relatively complex, we can just do:
```
std::vector<T> values;
std::vector<int32_t> row_splits;
row_splits.reserve(vecs.size() + 1);
int32_t tot_size = 0;
row_splits.push_back(tot_size);
for(const auto &vec: vecs){
    tot_size += static_cast<int32_t>(vec.size());
    row_splits.push_back(tot_size);
    values.insert(std::end(values), std::begin(vec), std::end(vec));
}
Array1<int32_t> row_splits_array(context, row_splits);
RaggedShape shape = RaggedShape2(&row_splits, nullptr, tot_size);
Array1<int32_t> values_array(context, values);
return Ragged<T>(shape, values_array);
```

Reply:
Yes, it's more concise.